blocks/footer/footer.css

```diff
@@ -11,3 +11,117 @@ footer .footer {
 footer .footer p {
   margin: 0;
 }
+
+.footer-container {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 8rem 0;
+}
+
+.footer-links-row {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.footer-links-pair {
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: row wrap;
+  flex: 1 1 280px;
+  position: relative;
+  padding: 1rem 0;
+}
+
+.footer-links-pair ul {
+  box-sizing: border-box;
+  letter-spacing: 0.025rem;
+  line-height: 1.6rem;
+  flex: 1 0 130px;
+  list-style-type: none;
+  margin: 0;
+  padding: .6rem 0;
+}
+
+.footer-links-pair li {
+  font-size: 1.2rem;
+  box-sizing: border-box;
+  margin-bottom: 1.6rem;
+}
+
+.footer-links-pair li:not(:has(a)) {
+  font-family: "Spezia_SemiMono", Arial, Helvetica, sans-serif;
```

Comment:
change this to use the var(--fixed-font-family) (you may need to correct the font selection in styles.css

Reply:
Done.